eng/validation/templates/testing/helix.yml

```diff
@@ -1,32 +1,27 @@
 jobs:
-  - job:
-    displayName: Validate Helix
-    variables:
-      - group: DotNet-HelixApi-Access
-      - name: _BuildConfig
-        value: Release
-    steps:      
-    - template: /eng/common/templates/steps/send-to-helix.yml
-      parameters:
-        HelixSource: pr/dotnet/arcade-validation/$(Build.SourceBranch)
-        HelixType: test/product/
-        HelixTargetQueues: Windows.81.Amd64.Open;Ubuntu.1604.Amd64.Open
-        XUnitProjects: $(Build.SourcesDirectory)/src/Validation/tests/Validation.Tests.csproj
-        XUnitTargetFramework: netcoreapp2.0
-        XUnitRunnerVersion: 2.4.1
-        IncludeDotNetCli: true
-        DotNetCliPackageType: sdk
-        DotNetCliVersion: 2.1.403
-        EnableXUnitReporter: true
-        WaitForWorkItemCompletion: true
-        IsExternal: true
-        Creator: arcade-validation
-        HelixAccessToken: $(HelixApiAccessToken)
-    - task: PublishBuildArtifacts@1
-      displayName: Publish Logs to VSTS
-      inputs:
-        PathtoPublish: '$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)'
-        PublishLocation: Container
-        ArtifactName: Validate-Helix
-      continueOnError: true
-      condition: always()
\ No newline at end of file
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      enablePublishBuildArtifacts: true
+      jobs:
+        - job:
+          displayName: Validate Helix
+          variables:
+            - group: DotNet-HelixApi-Access
```

Comment:
If we're using jobs.yml. We can use the variable:value syntax right?

Reply:
Hmm that's what I'm using bellow (?)

Reply:
You are indeed. Not sure what I was looking at 😮